### DISCUSSION TITLE
CouchDB Portable for windows

### DISCUSSION THREAD

**Discussion by DiegoJArg** [apache/couchdb#3040] [Show and tell] [answered]
A Portable distribution of binaries for windows is a common practice, as installers tend to require probably undesired and unknown changes. Also, it may be desirable for some users:

- To easily make snapshots.
- To use it from a pendrive while learning.
- As another way of offline support.
- To run it in user space, without OS administrator rights.
- To run tests or new codes outside a synced in-production installation.
- ...

As I was advised in an Issue posting:

> We don't advertise this, but you can use our downloadable MSI to install to a machine, then copy the entire installation folder somewhere else. It's 100% portable.
> 
> Just don't install the service, and beware of any folder permissions when copying that folder around.

So, I will expand here for future references.

- In windows don't install as a service.
- Go to Program Files and change Permissions for the Users group of all folders below CouchDB
- If you are in Windows 7, install IE11, or Fauxton wouldn't work.
- Run CouchDB from /bin/couchdb.cmd it will open the command line... if it closes then it has a problem.
- Open Fauxton, and choose Single Node type, and make the Admin user at least.
- Compress Apache CouchDB, or copy it into your pendrive.

**Comment by wohali** [comment, marked as answer]
OK.
